openair-collective/Carbon-Forming
# feature/#37-delete-flows

## Description of the change

Addresses issue #37 by adding deletion flows for Teams, Projects, and Competitions. I also fixed a few minor issues that have so far gone unreported. 

**Testing notes:**

1. Create Competition, Team, and Project models as needed
2. Delete buttons are found at the bottom of each models 'Edit' view
3. Deleting a Competition should also delete any associated Projects
4. Deleting a Team should also delete any associated Projects
5. Deleting a Project should just delete the project

## Type of change

### Code review 

- [x] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
